includes/discovery/sensors/temperature/beagleboard.inc.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/*
+ * Temperature Sensors for BeagleBoard
+ * Requires snmp extend agent script from librenms-agent
+ */
+
+use Illuminate\Support\Str;
+use LibreNMS\Config;
+
+$type = "beagleboardTemp";
+
+// $oid = 'NET-SNMP-EXTEND-MIB::nsExtendOutLine."beagleboard".';
+// Discovery is fine, but Polling seems to fail with string OID. So snmptranslate,
+//    snmptranslate -On NET-SNMP-EXTEND-MIB::nsExtendOutLine.\"beagleboard\"
+// to (with trailing ., to allow append in loop below),
+$oid = ".1.3.6.1.4.1.8072.1.3.2.4.1.2.11.98.101.97.103.108.101.98.111.97.114.100.";
+for ($sensor = 1; $volt <= 5; $sensor++) {
```

Comment:
$volt?

Reply:
Doh! 🤣. Copied this from Raspberry Pi, missed this one. Thanks!

Fixed, pushing updated files.